test/integration/active_model_searchable_test.rb

```diff
@@ -30,7 +30,10 @@ def teardown
         assert_equal 'czech', SupermodelArticle.mapping[:title][:analyzer]
         assert_equal 15,      SupermodelArticle.mapping[:title][:boost]
 
-        assert_equal 'czech', SupermodelArticle.index.mapping['supermodel_article']['properties']['title']['analyzer']
+        # This test seems to be incorrect, the #mapping response from ElasticSearch contains
+        # the results under the namespace 'mappings'
+        # assert_equal 'czech', SupermodelArticle.index.mapping['supermodel_article']['properties']['title']['analyzer']
+        assert_equal 'czech', SupermodelArticle.index.mapping['mappings']['supermodel_article']['properties']['title']['analyzer']
```

Comment:
It looks like ES changed the `mapping` response to include a top-level `mappings` namespace?

Reply:
I think I saw the same thing in Lucador at some point. See here: https://github.com/customink/luchador/pull/4